modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -11,6 +11,7 @@ InputParameters validParams<ComputeFiniteStrain>()
 {
   InputParameters params = validParams<ComputeStrainBase>();
   params.addClassDescription("Compute a strain increment and rotation increment for finite strains.");
+  params.set<bool>("stateful_displacements") = true;
```

Comment:
And what is the problem with copy paste here? This should just give a clear error.